environment

```diff
@@ -52,42 +50,54 @@ EXPORT_ENV_VARS_TO_LAMBDA_ARRAY=(
 set -a
 EXPORT_ENV_VARS_TO_LAMBDA=${EXPORT_ENV_VARS_TO_LAMBDA_ARRAY[*]}
 DSS_DEPLOYMENT_STAGE=dev
+STAGE="${DSS_DEPLOYMENT_STAGE}"
 ADMIN_USER_EMAILS_SECRETS_NAME=admin_user_emails
 AWS_DEFAULT_OUTPUT=json
-AWS_DEFAULT_REGION=us-east-1
+AWS_DEFAULT_REGION=us-west-2
 GCP_DEFAULT_REGION=us-central1
-GCP_PROJECT_NAME=human-cell-atlas-travis-test
+GCP_PROJECT_NAME="platform-hca"
 DSS_EVENT_RELAY_NAME=dss-gs-event-relay-
 CHECKOUT_CACHE_CRITERIA='[{"type":"application/json","max_size":12314}]'
-DSS_S3_BUCKET=org-humancellatlas-dss-dev
-DSS_S3_BUCKET_TEST=org-hca-dss-test
-DSS_S3_BUCKET_TEST_FIXTURES=org-hca-dss-test-fixtures
-DSS_S3_BUCKET_INTEGRATION=org-humancellatlas-dss-integration
-DSS_S3_BUCKET_STAGING=org-hca-dss-staging
-DSS_S3_BUCKET_PROD=org-humancellatlas-dss-prod
-DSS_S3_CHECKOUT_BUCKET=org-humancellatlas-dss-checkout-dev
-DSS_S3_CHECKOUT_BUCKET_TEST=org-hca-dss-checkout-test
-DSS_S3_CHECKOUT_BUCKET_TEST_USER=org-hca-dss-checkout-test-user  # for tests w/non-dss managed user bucket (i.e. POST /bundles/{uuid}/checkout)
-DSS_S3_CHECKOUT_BUCKET_UNWRITABLE=org-hca-dss-checkout-unwritable
-DSS_S3_CHECKOUT_BUCKET_INTEGRATION=org-humancellatlas-dss-checkout-integration
-DSS_S3_CHECKOUT_BUCKET_STAGING=org-hca-dss-checkout-staging
-DSS_S3_CHECKOUT_BUCKET_PROD=org-humancellatlas-dss-checkout-prod
-DSS_FLASHFLOOD_BUCKET=org-humancellatlas-dss-events-dev
-DSS_FLASHFLOOD_BUCKET_INTEGRATION=org-humancellatlas-dss-events-integration
-DSS_FLASHFLOOD_BUCKET_STAGING=org-humancellatlas-dss-events-staging
-DSS_FLASHFLOOD_BUCKET_PROD=org-humancellatlas-dss-events-prod
-DSS_GS_BUCKET=org-humancellatlas-dss-dev
-DSS_GS_BUCKET_TEST=org-hca-dss-test
-DSS_GS_BUCKET_TEST_FIXTURES=org-hca-dss-test-fixtures
-DSS_GS_BUCKET_INTEGRATION=org-humancellatlas-dss-integration
-DSS_GS_BUCKET_STAGING=org-hca-dss-staging
-DSS_GS_BUCKET_PROD=org-humancellatlas-dss-prod
-DSS_GS_CHECKOUT_BUCKET=org-humancellatlas-dss-checkout-dev
-DSS_GS_CHECKOUT_BUCKET_TEST=org-hca-dss-checkout-test
-DSS_GS_CHECKOUT_BUCKET_TEST_USER=org-hca-dss-checkout-test-user  # for tests w/non-dss managed user bucket (i.e. POST /bundles/{uuid}/checkout)
-DSS_GS_CHECKOUT_BUCKET_INTEGRATION=org-humancellatlas-dss-checkout-integration
-DSS_GS_CHECKOUT_BUCKET_STAGING=org-hca-dss-checkout-staging
-DSS_GS_CHECKOUT_BUCKET_PROD=org-humancellatlas-dss-checkout-prod
+
```

Comment:
we might want to introduce a new variable called platform, so we can perform interpolations like ${platform}-dss-checkout-bucket, or for a secret name ${platform}/dss/gcp-credentials.json.... by introducing a platform var, it lets the DSS become a little more flexible for different platforms it gets deployed to .

Reply:
agreed! I was thinking of adding a "project" variable but "platform" is a better name for it - less overloaded

Reply:
Setting `DSS_PLATFORM` to `ucsc-cgp` (for now at least), since this is a DSS deployment for internal UCSC use. Can change to something else if that is too confusing.

Reply:
implemented with 36b3eb0